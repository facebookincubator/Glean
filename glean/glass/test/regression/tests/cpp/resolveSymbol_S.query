action: resolveSymbolRange
args: "test/cpp//foo/S"
