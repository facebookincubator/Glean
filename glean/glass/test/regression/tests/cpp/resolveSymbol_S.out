[
    "@generated",
    {
        "filepath": "test.cpp",
        "range": {
            "columnBegin": 1,
            "columnEnd": 12,
            "lineBegin": 11,
            "lineEnd": 11
        },
        "repository": "test"
    }
]